### ISSUE TITLE
Incorrect group attribution

### ISSUE THREAD

**Issue by SlimeDog** [SrBedrock/PluginControl#31]
Spigot 1.20.2-3886
PluginControl 1.1.0

`Citizens` is not enabled (not yet updated to support MC 1.20.2). `pc reload` reports `Citizens` in the `constructor-builder-group`, which is not the case. The listing that follows is correct, per the configuration (appended).

```
>pc reload
[21:36:15] [Server thread/INFO]: [PluginControl] Config and Language reloaded!
[21:36:16] [Server thread/INFO]: [PluginControl] Checking plugins...
[21:36:16] [Server thread/INFO]: [PluginControl] Plugin Citizens, from the group constructor-builder-group not found or enabled...
>pc list
[21:36:35] [Server thread/INFO]: [PluginControl] Required plugins: Citizens
[21:36:35] [Server thread/INFO]: [PluginControl] Required groups: 
[21:36:35] [Server thread/INFO]: Group constructor-builder-group: [Constructor, ProSchematicBuilder]
```

config.yml
```
enabled: true
# Action to take if all listed plugins are not enabled.
# log-to-console
# disallow-player-login
# shutdown-server
action: disallow-player-login
plugins:
- Citizens
groups:
  constructor-builder-group:
  - Constructor
  - ProSchematicBuilder
```

**Comment by SrBedrock** [comment]
I couldn't reproduce this error, could you share the lang.yml file?

`config.yml`: 
```yaml
enabled: true
# Action to take if all listed plugins are not enabled.
# log-to-console
# disallow-player-login
# shutdown-server
action: disallow-player-login
plugins:
- Citizens
groups:
  constructor-builder-group:
  - ArmaBroadcast
  - Constructor
  - ProSchematicBuilder
```

Console:

![image](https://github.com/SrBedrock/PluginControl/assets/51332006/a3fdda17-515d-4d08-a250-0a00ec0de060)

**Comment by SlimeDog** [comment]
It looks like you added `ArmaBroadcast` to the group, which may not matter, but is different.

I posted just the relevant piece of config.yml above, but didn't test with that reduced plugin set -- my bad. I have a LOT of plugins.

Here are the complete files:
[config.yml](https://whitedog.straight8.com/drops/pc/config.yml)
[lang.yml](https://whitedog.straight8.com/drops/pc/lang.yml)

With the complete files, I get
```
[15:51:41] [Server thread/INFO]: [PluginControl] Plugin Citizens, from the group hologram-group, ItemJoin, from the group constructor-builder-group, pHD not found or enabled...
```
on server start-up. Note that `Citizens`, `ItemJoin`, and `pHD` are not loaded now, which is different from the previous report (`Citizens` had self-disabled). Neither `Citizens` or `ItemJoin` is a group member.

WRT lang.yml, I modified the `kick-message`, and removed the (MiniMessage) formatting codes. (I remove formatting codes for all plugins. I prefer to minimize the colors I see in my console, although I leave some.)

**Comment by SrBedrock** [comment]
Thanks for the detailed report, I believe I have found the problem, the order of the message that warns that no plugin from a group has been found is confusing, I will check which is the best alternative and change it.

**Comment by SrBedrock** [comment]
I've split the plugins and groups log to solve the problem, update your lang file and use the latest version to check if the problem has been solved.

new jar: https://github.com/SrBedrock/PluginControl/suites/16513036277/artifacts/943145236

new lang:
```yml
console:
  checking-plugins: '<prefix> <red>Checking plugins...'
  disabling-server: '<prefix> <red>Disabling server because plugins or groups were not found or enabled!'
  finished-checking: '<prefix> <green>Plugins verified!'
  log-to-console-plugin: '<prefix> <red>No plugins were found: <plugins>'
  log-to-console-group: '<prefix> <red>No groups were found: <groups>'
```

Full change: 

![image](https://github.com/SrBedrock/PluginControl/assets/51332006/1b6a315a-970f-4fd2-a44d-175098c5114b)

**Comment by SlimeDog** [comment]
Thanks. Please don't reuse version strings, at least for the release. Now I have multiple versions 1.1.0.

I recommend
```
  log-to-console-plugin: '<prefix> <red>Required plugins were not found: <plugins>'
  log-to-console-group: '<prefix> <red>Required groups were not found: <groups>'
```

**Comment by SlimeDog** [comment]
The output on server start-up looks great.
The output on `pc reload` looks equally great.
Thanks for fast response.

**Comment by SlimeDog** [comment]
Reminder: https://github.com/SrBedrock/PluginControl/issues/8 is still outstanding. I would understand if you choose not to do anything. In that case, just close the issue. Thanks.

**Comment by SrBedrock** [comment]
> Thanks. Please don't reuse version strings, at least for the release. Now I have multiple versions 1.1.0.

Noted, if I had published version 1.1.0 I would have changed the version to 1.1.1.

> I recommend
> 
> ```
>   log-to-console-plugin: '<prefix> <red>Required plugins were not found: <plugins>'
>   log-to-console-group: '<prefix> <red>Required groups were not found: <groups>'
> ```

Changed
